data/json/bionics.json

```diff
@@ -41,12 +41,21 @@
     "id": "bio_armor_arms",
     "type": "bionic",
     "name": { "str": "Alloy Plating - Arms" },
-    "description": "The flesh on your arms has been surgically replaced by alloy plating.  Provides passive protection and can be used in conjunction with bionic martial arts.",
-    "occupied_bodyparts": [ [ "arm_l", 4 ], [ "arm_r", 4 ] ],
-    "bash_protec": [ [ "arm_l", 3 ], [ "arm_r", 3 ] ],
-    "cut_protec": [ [ "arm_l", 3 ], [ "arm_r", 3 ] ],
-    "bullet_protec": [ [ "arm_l", 3 ], [ "arm_r", 3 ] ],
-    "flags": [ "BIONIC_NPC_USABLE", "BIONIC_SHOCKPROOF" ]
+    "description": "The skin and flesh on your arms has been surgically replaced by alloy plating.  Provides passive physical and chemical protection and can be used in conjunction with bionic martial arts at the cost of flexibility.",
+    "occupied_bodyparts": [ [ "arm_l", 5 ], [ "arm_r", 5 ] ],
+    "bash_protec": [ [ "arm_l", 7 ], [ "arm_r", 7 ] ],
+    "cut_protec": [ [ "arm_l", 5 ], [ "arm_r", 5 ] ],
+    "bullet_protec": [ [ "arm_l", 5 ], [ "arm_r", 5 ] ],
+    "fire_protec": [ [ "arm_l", 2 ], [ "arm_r", 2 ] ],
+    "acid_protec": [ [ "arm_l", 2 ], [ "arm_r", 2 ] ],
+    "env_protec": [ [ "arm_l", 2 ], [ "arm_r", 2 ] ],
+    "encumbrance": [ [ "arm_l", 4 ], [ "arm_r", 4 ] ],
+    "wet_protection": [ { "part": "arm_l", "ignored": 4 }, { "part": "arm_r", "ignored": 4 } ],
+    "covered_bodyparts": [ "arm_l", "arm_r" ],
+    "no_bleed_chance": 0.25,
+    "no_bite_chance": 0.33,
+    "cant_uninstal_msg": "The Alloy Plating is part of %s body now.  Removing it would leave %s disabled.",
+    "flags": [ "BIONIC_NPC_USABLE", "BIONIC_SHOCKPROOF", "BIONIC_CANT_UNINSTAL", "BIONIC_CANT_COMPRESS" ]
```

Comment:
```suggestion
    "cant_uninstall_msg": "The Alloy Plating is part of %s body now.  Removing it would leave %s disabled.",
    "flags": [ "BIONIC_NPC_USABLE", "BIONIC_SHOCKPROOF", "BIONIC_CANT_UNINSTALL", "BIONIC_CANT_COMPRESS" ]
```